Commands/DocumentSets/AddDocumentSet.cs

```diff
@@ -15,12 +15,14 @@ namespace SharePointPnP.PowerShell.Commands.DocumentSets
       SortOrder = 1)]
     public class AddDocumentSet : SPOWebCmdlet
     {
-        [Parameter(Mandatory = true)]
+        [Parameter(Mandatory = true, HelpMessage = "The list object where the document set needs to be added")]
```

Comment:
Better: The name of the list, its ID or an actual list object from where the document set needs to be added